features/bevragen/persoon/europees-kiesrecht/overzicht.feature

```diff
@@ -0,0 +1,150 @@
+    | burgerservicenummer | 000000127                       |
+    | fields              | europeesKiesrecht               |
+		Dan heeft de response een persoon met de volgende 'europeesKiesrecht' gegevens
+    | naam                    | waarde                 |
+    | aanduiding.code         | 1                      |
+    | aanduiding.omschrijving | persoon is uitgesloten |
+
+  Scenario: persoon is niet uitgesloten en ontvangt oproep
+		Gegeven de persoon met burgerservicenummer '000000127' heeft de volgende 'kiesrecht' gegevens
+    | Europees kiesrecht (31.10) |
+    | 2                          |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000127                       |
+    | fields              | europeesKiesrecht               |
+		Dan heeft de response een persoon met de volgende 'europeesKiesrecht' gegevens
+    | naam                    | waarde                  |
+    | aanduiding.code         | 2                       |
+    | aanduiding.omschrijving | persoon ontvangt oproep |
+
+Rule: Europees kiesrecht gegevens wordt geleverd als 'einddatum uitsluiting Europees kiesrecht (31.30)' in de toekomst ligt
+
+	Scenario: 'einddatum uitsluiting Europees kiesrecht (31.30)' ligt in de toekomst
+		Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'kiesrecht' gegevens
+    | Europees kiesrecht (31.10) | einddatum uitsluiting Europees kiesrecht (31.30) |
+    | 1                          | morgen                                           |
+		Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000164                       |
+    | fields              | europeesKiesrecht.aanduiding    |
+		Dan heeft de response een persoon met de volgende 'europeesKiesrecht' gegevens
+    | naam                    | waarde                 |
+    | aanduiding.code         | 1                      |
+    | aanduiding.omschrijving | persoon is uitgesloten |
+
+	Abstract Scenario: 'einddatum uitsluiting Europees kiesrecht (31.30)' is vandaag of een datum in het verleden
+		Gegeven de persoon met burgerservicenummer '000000164' heeft de volgende 'kiesrecht' gegevens
+    | Europees kiesrecht (31.10) | einddatum uitsluiting Europees kiesrecht (31.30) |
+    | 1                          | <einddatum uitsluiting>                          |
+		Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000164                       |
+    | fields              | europeesKiesrecht               |
+    Dan heeft de response een persoon zonder gegevens
```

Comment:
Is dat zo? Ter bevestiging: We leveren geen gegevens als de uitsluiting niet meer van toepassing is, tenzij code 2 en omschrijving (ontvangt oproep) gevuld zijn?

Reply:
dit is wel wat we in het verleden hebben afgesproken en overal (behalve voor partner) doen. Dus als de uitsluiting niet meer van toepassing is, communiceren we die niet.